src/constants.js

```diff
@@ -0,0 +1,6 @@
+// https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode
+export const INPUT_MODES = { NUMERIC: 'numeric' }
```

Comment:
Are we anticipating some more input modes? These always pretty much map to a value for [inputmode attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode) right?

The reason I ask above is I'm wondering if we should flatten this out if we don't really plan to use inputmodes for much else then kicking in numeric on mobile. For example, maybe we just make this `export const INPUT_NUMERIC = 'numeric'` so other devs don't have to mentally parse as much. Maybe same for the pattern. Thoughts?